### PR TITLE
🔀🌐[it_IT] vehicles

### PR DESCRIPTION
**Pull request by Alby78**

### CODE REVIEW

**Comment by KokosnussDEV** [comment]
Don't we already have translations for the file?

**Comment by JRH-1997** [comment]
Yes, it's already existing for it_IT

**Comment by Alby78** [comment]
> 
> 
> Don't we already have translations for the file?

I have translate the vheicles ;)

**Comment by KokosnussDEV** [comment]
You've actually translated the whole file

File: src/modules/missionHelper/i18n/it_IT.json [hunk 1]
```diff
@@ -0,0 +1,93 @@
+{
+	"diyMission": "Questa missione, sembra essere una missione dell\'Alleanza!",
```
Comment (Piet2001) [inline src/modules/missionHelper/i18n/it_IT.json:2]: @LSSM DEVS. Here a little issue. 
het \ has to be deleted. I think

**Comment by KBOE2** [comment]
Oh, we should update your fork next time @Alby78 
@LSS-Manager/developers please have a look on mergability 😅

File: src/modules/missionHelper/i18n/it_IT.json (hunk 1 above)
Comment (JRH-1997) [reply src/modules/missionHelper/i18n/it_IT.json:2]: Yes, because the "" the ' don't have to be escaped

**Comment by KBOE2** [comment]
Closing here as we first need to update the fork of @Alby78 before creating more conflicts.

**Pull request closed without merging**